cves/CVE-2015-2241.yml

```diff
@@ -165,8 +172,9 @@ subsystem:
     In the name field, a subsystem name (or an array of names)
 
     e.g. clipboard, model, view, controller, mod_dav, ui, authentication
-  answer:
-  name:
+  answer: |
+    The vulnerability was found in the admin section of the contributed code modules.
```

Comment:
Consider adding a path or maybe docs to this subsystem if there exists a documentation page.